keyboards/rabbit_capture_plan/paw3204.c

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2019 Sekigon
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "quantum.h"
+#include "paw3204.h"
```

Comment:
I suspect that a lot of this could be removed by using `spi_master`

Reply:
How do I use `spi_master`?

Reply:
https://docs.qmk.fm/#/spi_driver

You can also look at `/drivers/(avr|chibios)/spi_master.c`

Reply:
I deleted this file because I couldn't correct it well.